assets/src/components/AssignmentTable.js

```diff
@@ -382,13 +437,15 @@ function AssignmentTable (props) {
                           </div>
                         </TableCell>
                         <TableCell className={classes.veryNarrowCell}>
-                          {a.graded ? <DoneIcon /> : <div />}
+                          <Tooltip title={a.graded ? assignmentStatus.GRADED : a.submitted ? assignmentStatus.SUBMITTED : assignmentStatus.UNSUBMITTED}>
+                            {a.graded ? <GradedIcon className={classes.graded} /> : a.submitted ? <SubmittedIcon className={classes.ungraded} /> : <UnsubmittedIcon className={classes.unsubmitted} />}
+                          </Tooltip>
```

Comment:
Where in the UI this little block is used?

Reply:
Tooltip for the icon displayed in the Status column